### PR TITLE
Issue #125: [addition] - Added drawPoint, drawArc, and drawBezier

### PR DESCRIPTION
**Pull request by ishubham21**
- add function to draw a point
- add function to draw an arc
- add function to draw a bezier curve
- add function to set fill on
- add function to set fill off

### CODE REVIEW

File: src/components/painter/sketchP5.ts [hunk 1]
```diff
@@ -36,6 +36,53 @@ export function drawLine(x1: number, y1: number, x2: number, y2: number): void {
...
+ * Draws a point.
+ * @param x - x co-ordinate of the point
+ * @param y - y co-ordinate of the point
+ */
+export function drawPoint(x: number, y: number): void {
+    sketch.point(x, y);
+}
+
+/**
+ * Draws a arc.
```
Comment (meganindya) [inline src/components/painter/sketchP5.ts:49]: *an arc :p

File: src/components/painter/sketchP5.ts [hunk 2]
```diff
@@ -36,6 +36,53 @@ export function drawLine(x1: number, y1: number, x2: number, y2: number): void {
...
+ * @param start - angle to start the arc - in Radians
+ * @param stop - angle to stop the arc - in Radians
+ */
+export function drawArc(
+    x: number,
+    y: number,
+    w: number,
+    h: number,
+    start: number,
+    stop: number): void {
```
Comment (meganindya) [inline src/components/painter/sketchP5.ts:63]:
```ts
export function drawArc(
    x: number,
    y: number,
    w: number,
    h: number,
    start: number,
    stop: number,
): void {
```

this is the style convention for JS/TS

File: src/components/painter/sketchP5.ts [hunk 3]
```diff
@@ -36,6 +36,53 @@ export function drawLine(x1: number, y1: number, x2: number, y2: number): void {
...
+ * @param x3 - x co-ordinate of control point 2
+ * @param y3 - y co-ordinate of control point 2
+ * @param x4 - x co-ordinate of anchor point 2
+ * @param y4 - y co-ordinate of anchor point 2
+ */
+export function drawBezier(
+    x1: number, y1: number,
+    x2: number, y2: number,
+    x3: number, y3: number,
+    x4: number, y4: number): void {
```
Comment (meganindya) [inline src/components/painter/sketchP5.ts:82]:
```ts
export function drawBezier(
    x1: number,
    y1: number,
    x2: number,
    y2: number,
    x3: number,
    y3: number,
    x4: number,
    y4: number,
): void {
```

File: src/components/painter/sketchP5.ts [hunk 4]
```diff
@@ -36,6 +36,53 @@ export function drawLine(x1: number, y1: number, x2: number, y2: number): void {
...
+ * @param stop - angle to stop the arc - in Radians
+ */
+export function drawArc(
+    x: number,
+    y: number,
+    w: number,
+    h: number,
+    start: number,
+    stop: number): void {
+    sketch.arc(x, y, w, h, start, stop);
```
Comment (meganindya) [inline src/components/painter/sketchP5.ts:64]:
```ts
sketch.arc(x, y, w, h, start, stop, sketch.OPEN);
```

explicitly add the `mode`

**Review by meganindya — CHANGES_REQUESTED** [review]
Just make the minor changes. Otherwise LGTM.

BTW, please update your commit message to `painter: [p5] add more functions` and description to
```text
- add function to draw a point
- add function to draw an arc
- add function to draw a bezier curve
```

**Comment by meganindya** [comment]
You can add the `fill` and `noFill` functions too. Those are identical to the `p5.fill()` and `p5.noFill()` functions

**Comment by ishubham21** [comment]
@meganindya I have updated the PR with the requested changes, please review.

**Review by meganindya — APPROVED** [review]
Good work. Thanks!

**Pull request merged**